src/components/flag/Flag.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import Flag from "./Flag";
+
+export default {
+  title: "Components/Flag",
+  component: Flag,
+  argTypes: {
+    size: {
+      name: "size",
+      description: "Size of the icon",
+      control: { type: "number" },
+      type: "number"
+    },
+    countryCode: {
+      name: "countryCode",
+      description: "Country code",
+      control: { type: "text" },
+      type: "string"
+    },
+  },
+} as ComponentMeta<typeof Flag>;
+
+const Template: ComponentStory<typeof Flag> = (args) => <Flag  {...args} />;
+
+export const Flag_12 = Template.bind({});
```

Comment:
You can keep only one version, then the arguments can be used in storybook to test the variations.

Reply:
Like in https://league-q6roqqxfe-caisy.vercel.app/?path=/story/components-button--default